From b1710e40cacdc5d4b10ff313ba17548d22114db7 Mon Sep 17 00:00:00 2001
From: lijian <jian.li@freescale.com>
Date: Fri, 27 Jun 2014 12:02:09 +0800
Subject: [PATCH 329/602] ENGR00319972 correct the libav library name for user
 build

-   libav codec and component libraries are not installed
    into user build image, cause all playback failed.

Signed-off-by: lijian <jian.li@freescale.com>
---
 imx5x/imx5x.mk | 6 +++---
 imx6/imx6.mk   | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/imx5x/imx5x.mk b/imx5x/imx5x.mk
index 17e2b2e..bb8fcd5 100644
--- a/imx5x/imx5x.mk
+++ b/imx5x/imx5x.mk
@@ -240,9 +240,9 @@ omx_libs := \
 	lib_wav_parser_arm11_elinux.3.0 \
 	lib_omx_ac3toiec937_arm11_elinux \
         lib_omx_ec3_dec_v2_arm11_elinux \
-		lib_omx_libav_video_dec_arm11_elinux.so \
-        libavcodec-55.so \
-        libavutil-53.so \
+	lib_omx_libav_video_dec_arm11_elinux \
+	libavcodec-55 \
+	libavutil-53 \
 
 
 
diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 524eb57..18af659 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -259,9 +259,9 @@ omx_libs := \
 	lib_wav_parser_arm11_elinux.3.0 \
 	lib_omx_ac3toiec937_arm11_elinux \
         lib_omx_ec3_dec_v2_arm11_elinux \
-		lib_omx_libav_video_dec_arm11_elinux.so \
-        libavcodec-55.so \
-        libavutil-53.so \
+	lib_omx_libav_video_dec_arm11_elinux \
+	libavcodec-55 \
+	libavutil-53 \
 
 
 
-- 
1.8.0

